README.md

```diff
@@ -1,12 +1,17 @@
 # Active Travel Infrastructure Platform
 
-ATIP lets you sketch new plans for walking and cycling infrastructure with a
-low level of detail. See the initial version at
-<https://acteng.github.io/atip>.
+ATIP lets you design, modify and share active travel infrastructure plans.
+
+<!-- If we want to say more (less can be more): -->
+<!-- It is an open source web application, meaning anyone can use it in their browser.
+It developed by the Alan Turing Institute in collaboration with Active Travel England.
+ATIP reduces barriers to entry into the transport planning process, and reduces friction between different stakeholders. -->
 
 ## Using ATIP
 
-If you're in England, go use the development version at <https://acteng.github.io/atip>. Note this is not a production service; we make no guarantees about the site's uptime or about backwards compatibility of ATIP files.
+The development version of ATIP is available for use at <https://acteng.github.io/atip>.
+Note: this is not a production service and we make no guarantees about the site's uptime or about backwards compatibility of ATIP files.
+The route snapping functionality in that version only works in England.
```

Comment:
I don't think we need this last sentence; the dev version linked only shows England. There's a section below about running elsewhere.

Reply:
Good point will remove.

Reply:
Tidied up, now only one mention of this.